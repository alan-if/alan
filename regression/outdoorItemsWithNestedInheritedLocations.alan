The outdoor Isa location
End The outdoor.

Every outdoorScenery Isa object At outdoor
  Verb x
    Does
      Say The This.
  End Verb.
End Every outdoorScenery.

The sky Isa outdoorScenery
End The sky.

The ground Isa outdoorScenery
End The ground.

Every outdoorLocation Isa location At outdoor
End Every outdoorLocation.

The l1 Isa outdoorLocation
  Exit e To l2.
End The l1.

The l2 Isa location
  Exit w To l1.
End The l2.

Syntax x = x (o).

Start At l1.
