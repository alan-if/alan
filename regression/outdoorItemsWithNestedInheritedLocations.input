x sky
x ground
e
x sky
x ground
