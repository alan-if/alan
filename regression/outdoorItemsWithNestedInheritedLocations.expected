########## outdoorItemsWithNestedInheritedLocations ##########

        No warnings or errors detected.



L1

> x sky
The sky

> x ground
The ground

> e
L2

> x sky
I can't see any sky here.

> x ground
I can't see any ground here.

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
